templates/withKibana/accounts.go

```diff
@@ -0,0 +1,26 @@
+package withKibana
+
+import (
+	"github.com/ElrondNetwork/elastic-indexer-go/templates"
+)
+
+type Object = templates.Object
```

Comment:
Also here rename is not really necessary, right?

Reply:
Is not really necessary. I used this approach in order to use `Object` type without using his package name